pipeline/metadata/satellite.py

```diff
@@ -421,13 +467,19 @@ def process_satellite_with_tags(
   rows = (
       row_lines | 'flatten json' >> beam.ParDo(
           flatten.FlattenMeasurement()).with_output_types(Row))
+
+  # PCollection[Row] each with only a single element in the received arrays
+  received_ip_flattened_rows = (
+      rows | 'flatten received ips' >>
```

Comment:
This is where you can create the two separate PCollections by running two different transforms on the `rows`.